jobs/pull-test-infra-bazel.sh

```diff
@@ -20,6 +20,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+pip install -r experiment/requirements.txt
```

Comment:
I'm not familiar with the external workspace rule. Could you elaborate? Or is there an example I could follow?